mt32emu/src/mt32emu.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/include/mt32emu
+
+Name: libmt32emu
+Description: a C/C++ library which allows to emulate (approximately) the Roland MT-32, CM-32L and LAPC-I synthesiser modules
+Version: @libmt32emu_VERSION@
+Libs: -L${libdir} -lmt32emu
+Cflags: -I${includedir}
```

Comment:
Hmm, one more thing. pkg-config-guide.html shows in the example
`Cflags: -I${includedir}/foo`
so we probably should move the subdir here from includedir, wdyt?